Removed registry deployment from taskruns

sajayantony opened the pull request:
Signed-off-by: Sajay Antony <sajaya@microsoft.com>

northtyphoon commented:
The sample was originally to show an example to deploy a registry and taskrun from scratch. As the default mode is incremental, users can tweak the taskrun settings and retry the deployment without worrying about recreating the same registry. Did you see any issue?

sajayantony commented:
Added the E2E back and also added it to the docs page. 
![image](https://user-images.githubusercontent.com/1821104/69193856-98bfa200-0adc-11ea-8caa-af8d00b56c4d.png)
